src/test/kotlin/org/jitsi/jibri/api/http/internal/InternalHttpApiTest.kt

```diff
@@ -46,20 +46,33 @@ class InternalHttpApiTest : ShouldSpec() {
                 val gracefulShutdownHandler = {
                     gracefulShutdownHandlerCalled = true
                 }
-                val internalHttpApi = InternalHttpApi(gracefulShutdownHandler, {})
+                val internalHttpApi = InternalHttpApi({}, gracefulShutdownHandler, {})
                 val response = internalHttpApi.gracefulShutdown()
                 response.status shouldBe 200
                 gracefulShutdownHandlerCalled shouldBe false
             }
         }
 
+        "notifyConfigChanged" {
+            should("return a 200 and not invoke the config changed handler directly") {
+                var configChangedHandlerCalled = false
+                val configChangedHandler = {
+                    configChangedHandlerCalled = true
+                }
+                val internalHttpApi = InternalHttpApi({}, configChangedHandler, {})
+                val response = internalHttpApi.reloadConfig()
+                response.status shouldBe 200
+                configChangedHandlerCalled shouldBe false
```

Comment:
realizing now that these tests should really verify that the handler does _eventually_ get called...but i'll go in and fix that later